<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>LocationDialog</class>
 <widget class="QWidget" name="LocationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="CityGroupBox" >
     <property name="title" >
      <string>Choose City</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="MapCanvas" name="MapView" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>360</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>360</width>
              <height>180</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="0" >
           <widget class="QLabel" name="CountryFiltLabel" >
            <property name="text" >
             <string>Country filter:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KLineEdit" name="ProvinceFilter" />
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="CityFiltLabel" >
            <property name="text" >
             <string>City filter:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="KLineEdit" name="CountryFilter" />
          </item>
          <item row="0" column="1" >
           <widget class="KLineEdit" name="CityFilter" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="ProvinceFiltLabel" >
            <property name="text" >
             <string>Province filter:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="CountLabel" >
          <property name="text" >
           <string>0 cities match search criteria</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="KListBox" name="GeoBox" >
        <property name="vScrollBarMode" >
         <enum>Q3ScrollView::Auto</enum>
        </property>
        <property name="hScrollBarMode" >
         <enum>Q3ScrollView::Auto</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="EditGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>View/Edit Location Data</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="1" >
         <widget class="KLineEdit" name="NewProvinceName" >
          <property name="minimumSize" >
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="KLineEdit" name="NewCountryName" >
          <property name="minimumSize" >
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="NewCityLabel" >
          <property name="text" >
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="KLineEdit" name="NewCityName" >
          <property name="minimumSize" >
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="NewProvinceLabel" >
          <property name="text" >
           <string>Province:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="NewCountryLabel" >
          <property name="text" >
           <string>Country:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="LongLabel" >
              <property name="text" >
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="dmsBox" name="NewLong" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="dmsBox" name="NewLat" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="LatLabel" >
              <property name="text" >
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="0" >
             <widget class="K3ActiveLabel" name="DSTLabel" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <family>Sans Serif</family>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
                <underline>true</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text" >
               <string>DST rule:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QComboBox" name="DSTRuleBox" />
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="TZLabel" >
              <property name="text" >
               <string>UT offset:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QComboBox" name="TZBox" >
              <property name="editable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="KPushButton" name="ClearFieldsButton" >
            <property name="text" >
             <string>&amp;Clear Fields</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="KPushButton" name="AddCityButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Add City to List</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>MapCanvas</class>
   <extends>QFrame</extends>
   <header>widgets/mapcanvas.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends></extends>
   <header>klineedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>K3ActiveLabel</class>
   <extends></extends>
   <header>k3activelabel.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KListBox</class>
   <extends></extends>
   <header>klistbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends></extends>
   <header>kpushbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CityFilter</tabstop>
  <tabstop>ProvinceFilter</tabstop>
  <tabstop>CountryFilter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
